packages/ui/core-components/src/lib/atoms/inputs/dropdown/Dropdown.svelte

```diff
@@ -346,7 +356,13 @@
 						<Command.List>
 							<Command.Empty>No results found.</Command.Empty>
 							<Command.Group>
-								<VirtualList height="160px" items={$options} let:item={option}>
+								<!-- kwongz height handle by function, passing Displayed_Options to component for handling overflow (scroll) -->
+								<VirtualList
+									height={handleDropDownHeight($options, DISPLAYED_OPTIONS)}
+									displayedOptions={DISPLAYED_OPTIONS}
```

Comment:
I don't know if we want VirtualList to be aware of this - it shouldn't be responsible for knowing about this